benchmark/single-source/NSDictionaryCastToSwift.swift

```diff
@@ -20,6 +20,7 @@ import TestsUtils
 
 @inline(never)
 public func run_NSDictionaryCastToSwift(_ N: Int) {
+#if _runtime(_ObjC)
```

Comment:
It is probably better if practical to just #if out the entire functions if possible, and then just not call them on Linux, rather than call them with empty bodies.